integrations/datadog/.port/resources/blueprints.json

```diff
@@ -1,4 +1,63 @@
 [
+  {
+    "identifier": "datadogHost",
+    "description": "This blueprint represents a datadog host",
+    "title": "Datadog Host",
+    "icon": "Datadog",
+    "schema": {
+      "properties": {
+        "up": {
+          "type": "boolean",
+          "title": "Is Running?",
+          "description": "Is the host up?"
+        },
+        "hostName": {
+          "type": "string",
+          "title": "Host Name",
+          "description": "the name of the host"
+        },
+        "description": {
+          "type": "string",
+          "title": "Description",
+          "description": "the host description"
+        },
+        "platform": {
+          "type": "string",
+          "title": "Platform",
+          "description": "the host platform"
+        },
+        "machine": {
+          "type": "string",
+          "title": "Machine",
+          "description": "The CPU architecture of the host machine (e.g., amd64, x86, x64, arm, arm64)."
+        },
+        "cpuCores": {
+          "type": "number",
+          "title": "CPU Cores",
+          "description": "the host CPU cores"
+        },
+        "agent_version": {
+          "type": "string",
+          "title": "Datadog Agent Version",
```

Comment:
please be consistent with the properties identifiers lets use underscore, don't forget to update mapping as well

Reply:
alright